lib/deps/binary/base64.js

```diff
@@ -4,10 +4,10 @@ var buffer = require('./buffer');
 
 exports.atob = function (str) {
   var base64 = new buffer(str, 'base64');
-  // Node.js will just skip the characters it can't encode instead of
-  // throwing and exception
+  // Node.js will just skip the characters it can't decode instead of
+  // throwing an exception
   if (base64.toString('base64') !== str) {
-    throw ("Cannot base64 encode full string");
+    throw 'InvalidBase64Error';
```

Comment:
Oh I see, it gets discarded no matter what. Then yeah, it doesn't matter what we throw; it will never get used. I would still say `throw new Error("same string we use elsewhere")`, since gzip will compact that string down anyway, and it's helpful to have a message in case we ever need to debug.